modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -95,60 +95,64 @@ The API that was created is displayed.
 
 Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials.
 
+*Task Prerequisites*
```

Comment:
The thing I don't like about using this as an inline heading is that it is a heading for only one item and, more importantly, there is no indication that the actual task begins next.
Are the steps that follow how you MEET the prerequisites?
It's unclear.